configure

```diff
@@ -214,6 +214,10 @@ def _load_template_mapping():
     return (len(messages) == 0, messages)
 
 
+def _clean(val):
+    return val.split("#")[0].strip()
```

Comment:
Maybe a docstring just to describe what this is operating on.  It's pretty clear what it does, maybe less clear what will be fed to it.